flink-ml-lib/src/main/java/org/apache/flink/ml/operator/common/LinearModelType.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.ml.operator.common;
+
+/** Types of linear model types. */
+public enum LinearModelType {
```

Comment:
It is not clear to me what we need to have this enum type. Can we add this only when it is needed?

Reply:
Thanks for the review!

The enum type is used for different linear models, such as Logistic Regression, Linear Regression and SVM, etc.

Of course, we can add them later. I will add more linear models to demonstrate this in this PR.

Reply:
As discussed offline, we aggreed to add this later.